docs/api/zod/coerceFormValue.md

```diff
@@ -162,7 +145,6 @@ const schema = coerceFormValue(
         };
       }
 
-      // Return `null` to keep the default behavior
```

Comment:
I thought it would be fine to leave this comment, but should I delete it?

Of course, returning `null` will result in the behavior defined by the `defaultCoercion` option or the default behavior, so would it be easier to understand if you made that clear?